enterprise/internal/campaigns/testdata/fixtures/webhooks/bitbucketserver/add-review.json

```
@@ -0,0 +1,476 @@
+                    {
+                      "href": "https://bitbucket.sgdev.org/users/thorsten"
+                    }
+                  ]
+                },
+                "avatarUrl": "https://secure.gravatar.com/avatar/3019a6cedc57148130de2a9c97977eb3.jpg?s=64\u0026d=mm"
+              },
+              "role": "AUTHOR",
+              "approved": false,
+              "status": "UNAPPROVED"
+            },
+            "reviewers": [
+              {
+                "user": {
+                  "name": "milton",
+                  "emailAddress": "dev@sourcegraph.com",
+                  "id": 1,
+                  "displayName": "milton woof",
+                  "active": true,
+                  "slug": "milton",
+                  "type": "NORMAL",
+                  "links": {
+                    "self": [
+                      {
+                        "href": "https://bitbucket.sgdev.org/users/milton"
+                      }
+                    ]
+                  },
+                  "avatarUrl": "/users/milton/avatar.png?s=64\u0026v=1576525349000"
+                },
+                "role": "REVIEWER",
+                "approved": false,
+                "status": "UNAPPROVED"
+              }
+            ],
+            "participants": [],
+            "links": {
+              "self": [
+                {
+                  "href": "https://bitbucket.sgdev.org/projects/SOUR/repos/automation-testing/pull-requests/19"
+                }
+              ]
+            }
+          },
+          "activity": {
+            "id": 3003,
+            "createdDate": 1588170844885,
+            "user": {
+              "name": "milton",
+              "emailAddress": "dev@sourcegraph.com",
+              "id": 1,
+              "displayName": "milton woof",
+              "active": true,
+              "slug": "milton",
+              "type": "NORMAL",
+              "links": {
+                "self": [
+                  {
+                    "href": "https://bitbucket.sgdev.org/users/milton"
+                  }
+                ]
+              },
+              "avatarUrl": "/users/milton/avatar.png?s=64\u0026v=1576525349000"
+            },
+            "action": "UPDATED",
+            "addedReviewers": [
+              {
+                "name": "milton",
+                "emailAddress": "dev@sourcegraph.com",
+                "id": 1,
+                "displayName": "milton woof",
+                "active": true,
+                "slug": "milton",
+                "type": "NORMAL",
+                "links": {
+                  "self": [
+                    {
+                      "href": "https://bitbucket.sgdev.org/users/milton"
+                    }
+                  ]
+                },
+                "avatarUrl": "/users/milton/avatar.png?s=64\u0026v=1576525349000"
+              }
+            ],
+            "removedReviewers": []
+          }
+        }
+      },
+      {
+        "payload_type": "",
```

It's a bit confusing seeing this empty string here, so I'd suggest either omitting it or adding a dummy value, maybe:

```suggestion
        "payload_type": "IGNORED_FOR_THIS_EVENT",
```